src/components/graphs/DualAnnualCycleGraph.js

```diff
@@ -127,10 +127,16 @@ export default function DualAnnualCycleGraph(
       const variableOverlaps = getVariableOptions(variable_id, 'shiftAnnualCycle');
       const comparandOverlaps = getVariableOptions(comparand_id, 'shiftAnnualCycle');
       
+      // see if both variables are annual-only, in which case they visually
+      // overlap as well, because both will be graphed as horizontal lines.
+      //const annualOnly = !_.contains(_.pluck(meta, "timescale"), "seasonal")
+      //&& !_.contains(_.pluck(meta, "timescale"), "monthly");
+      const annualOnly = (_.reject(meta, m => {return m.timescale === 'yearly';})).length === 0;
```

Comment:
Tiny little tweak: There's a shorthand for an arrow function that consists of a single return statement:

```
m => m.timescale === 'yearly'
```

and it's also possible that underscore has a shorthand so that you don't have to construct this function.